src/components/UserForm.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Container, Title } from '@mantine/core';
+import useStateWithLocalStorage from './UseStateWithLocalStorage';
+
+const UserForm = () => {
+  const [inputValue, setInputValue] = useStateWithLocalStorage('', 'form');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
```

Comment:
Currently does not do anything as the form is never submitted, can be removed

Reply:
haha yeh I deleted the button